### PR TITLE
Added docs to the  folder

### PR DESCRIPTION
**Pull request by alwasega**
@minrk, @sgibson91, and everyone, 

I have added the three files under `Tutorials` [here](https://docs.google.com/spreadsheets/d/1Nnwji40YXMklnQ2yvO3rcfGnMP8S_LsVlkKdobo8ZjM/edit#gid=307391957) to the Tutorials folder in the repository. In doing so, I have deleted the old `Installation Guide` folder as it contained only the three moved docs.  

I have also slightly edited the top-level index file as well as the index file within the new Tutorials folder in the repository. 

Please let me know if any changes are necessary.

### CODE REVIEW

**Comment by minrk** [comment]
There are a couple of relative-path links that have broken:

```
/home/runner/work/jupyterhub/jupyterhub/docs/source/tutorial/installation/installation-basics.md:5: WARNING: 'myst' reference target not found: ./getting-started/security-basics
[179](https://github.com/jupyterhub/jupyterhub/pull/4305/checks#step:6:180)
writing output... [100%] tutorial/installation/quickstart-docker               
[180](https://github.com/jupyterhub/jupyterhub/pull/4305/checks#step:6:181)
/home/runner/work/jupyterhub/jupyterhub/docs/source/tutorial/installation/quickstart.md:29: WARNING: 'myst' reference target not found: ./getting-started/authenticators-users-basics.md
```

These should probably be updated to just myst references, instead of taking paths into account, since that will make them stable across future reorganization.

**Comment by alwasega** [comment]
Thanks @minrk. 
However, I don't quite understand your response. Is there something I can do on my end to resolve the issue?

**Comment by sgibson91** [comment]
@alwasega Yes. Let's talk about this in our meeting on Monday and I can show you how to resolve this.

**Comment by alwasega** [comment]
> Let's talk about this in our meeting on Monday and I can show you how to resolve this.

Great. Thank you!

File: docs/source/getting-started/security-basics.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Security settings
+(security-basics)=
```
Comment (sgibson91) [inline docs/source/getting-started/security-basics.md:3]: Same here, put the anchor above the title :)

File: docs/source/tutorial/installation/installation-basics.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ JupyterHub is supported on Linux/Unix based systems. To use JupyterHub, you need
 a Unix server (typically Linux) running somewhere that is accessible to your
 team on the network. The JupyterHub server can be on an internal network at your
 organization, or it can run on the public internet (in which case, take care
-with the Hub's [security](./getting-started/security-basics)).
+with the Hub's [security](security-basics).
```
Comment (sgibson91) [inline docs/source/tutorial/installation/installation-basics.md:9]: There is an open parenthesis just before "in which case", so we need two closing parentheses here :)

File: docs/source/getting-started/authenticators-users-basics.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Authentication and User Basics
 
+(authenticators-users-basics)=
```
Comment (sgibson91) [inline docs/source/getting-started/authenticators-users-basics.md:3]:
```
/home/runner/work/jupyterhub/jupyterhub/docs/source/getting-started/config-basics.md:81: WARNING: more than one target found for 'myst' cross-reference authenticators-users-basics: could be :std:ref:`authenticators` or :std:doc:`authenticators` [myst.ref]
```

This warning means that the reference `authenticators-users-basics` is already in use in another file, `config-basics.md`. So maybe we could change this reference to `authenticators-getting-started`?

[the remark this answers is not in the document]
Comment (sgibson91) [reply docs/source/getting-started/authenticators-users-basics.md:3]: I am no longer sure that this matters after inspecting the current error message

Comment (alwasega) [reply docs/source/getting-started/authenticators-users-basics.md]: Sorry, I changed it to just `authenticators` before seeing this comment. Can that work? I see all tests now pass?

Comment (sgibson91) [reply docs/source/getting-started/authenticators-users-basics.md]: Yes, so long as it is a unique reference, it will work :)

**Comment by alwasega** [comment]
Thank you @sgibson91!

**Pull request merged**